pyabc/visualization/credible.py

```diff
@@ -144,18 +152,24 @@ def plot_credible_intervals(
                 y=median[i_par].flatten(),
                 yerr=[median[i_par] - cis[i_par, :, i_c],
                       cis[i_par, :, -1 - i_c] - median[i_par]],
+                color = colors[i_c],
+                ecolor = colors[i_c],
                 capsize=(5.0 / n_confidence) * (i_c + 1),
                 label="{:.2f}".format(confidence))
         ax.set_title(f"Parameter {par}")
         # mean
+        color_index = len(levels)
```

Comment:
ah, you use the remaining colors for show_mean etc. I think it may be better to define separate arguments `color_mean`, `color_kde_max` etc.

Reply:
Ok, will adjust that